extensions/sql-migration/src/wizard/skuRecommendationPage.ts

```diff
@@ -243,16 +259,21 @@ export class SKURecommendationPage extends MigrationWizardPage {
 		}
 	}
 
-	private async populateManagedInstanceDropdown(): Promise<void> {
-		if (!this.migrationStateModel._targetManagedInstance) {
-			this._managedInstanceDropdown.loading = true;
-			try {
-				this._managedInstanceDropdown.values = await this.migrationStateModel.getManagedInstanceValues(this.migrationStateModel._targetSubscription);
-			} catch (e) {
-				console.log(e);
-			} finally {
-				this._managedInstanceDropdown.loading = false;
+	private async populateResourceInstanceDropdown(): Promise<void> {
+		this._resourceDropdown.loading = true;
+		try {
+			if (this._rbg.selectedCardId === 'AzureSQLVM') {
+				this._resourceDropdownLabel.value = constants.AZURE_SQL_DATABASE_VIRTUAL_MACHINE;
+				this._resourceDropdown.values = await this.migrationStateModel.getSqlVirtualMachineValues(this.migrationStateModel._targetSubscription);
+
+			} else {
+				this._resourceDropdownLabel.value = constants.AZURE_SQL_DATABASE_MANAGED_INSTANCE;
+				this._resourceDropdown.values = await this.migrationStateModel.getManagedInstanceValues(this.migrationStateModel._targetSubscription);
 			}
+		} catch (e) {
+			console.log(e);
```

Comment:
Don't want to display this to the user? They might find it useful to know when an error happened so they can try to do something about it